profiles/base.yaml

```diff
@@ -193,6 +193,7 @@ crossing_training_requires_movement:
 - Trading
 forage_item:
 braid_item:
+collect_item: rock
```

Comment:
It seems reasonable to me to, as part of this PR, make use of this setting in more places.